In [1]:
it = 0
arr = []
with open('tryRF_mss3.txt') as f:
    for line in f:
        it += 1
        if (it > 1):
            arr += [line[:-1].split(' ')]

In [2]:
minimumm = 10.
ind = 0
for h in range(len(arr)):
    if (float(minimumm) > float(arr[h][3])):
        minimumm = arr[h][3]
        ind = h
print minimumm
print ind
print arr[ind]
arr[ind][3] = 100.
minimumm = 10.
ind = 0
for h in range(len(arr)):
    if (float(minimumm) > float(arr[h][3])):
        minimumm = arr[h][3]
        ind = h
print minimumm
print ind
print arr[ind]

0.267552400686
89
['17', '10', '9', '0.267552400686']
0.268939095788
179
['20', '10', '9', '0.268939095788']


1 . RandomForest, 
lb_smape = 28, my_smape = 0.30866, 
parametrs : (num_est, c, a, smape)
['18', '9', '5', '0.308662558109']


3 . Bagging, 
lb_smape = 29, my_smape = 0.30869, 
parametrs : (num_est, c, a, smape)
['18', '9', '5', '0.308699860161']


5 . RandomForest (mss = 3, num_est = 17),
lb_smape = 25, my_smape = 0.26755, 
parametrs : (num_est, c, a, smape)
['17', '10', '9', '0.267552400686']


6 . RandomForest (mss = 3, num_est = 17 + ohe),
lb_smape = 23.13